os_faults/drivers/tcpcloud.py

```diff
@@ -177,6 +177,35 @@ class NovaSpiceService(SaltService):
     GREP = '[n]ova-spicehtml5proxy'
     SALT_SERVICE = 'nova-spicehtml5proxy'
 
+class NeutronMeteringAgentService(SaltService):
```

Comment:
Add these to line 372 in this file.  I just realized I missed that for all of my new services